book/src/slasher.md

```diff
@@ -189,6 +192,6 @@ lighthouse bn --slasher --slasher-history-length 256 --slasher-max-db-size 16 --
 ```
 
 ## Stability Warning
-
+[comment]: <> (Not sure if this subsection should be updated? Like not sure about the "quite new" description.)
```

Comment:
```suggestion
```

Oh yeah, good point. I think we can leave it as-is for now. We could maybe change "is still quite new" to "has not yet ossified". However I don't feel strongly about that and could happily leave it as-is.